### ISSUE TITLE
Use the stdlib instead of atty

### ISSUE THREAD

**Issue by o0Ignition0o** [iomentum/cargo-scaffold#43]
https://github.com/apollographql/router/issues/3809

The apollo-router-scaffold crate currently depends on clap version 2.34.0 which depends on the atty crate which [has potential unsoundness and is unmaintained](https://github.com/advisories/GHSA-g98v-hv3f-hcfr). Upgrading clap to version 4.4.2 (same as the apollo-router depends on) would remove atty from the workspace entirely.

**Comment by gabhijit** [comment]
@o0Ignition0o : In my local fork, I quickly tried using `clap` version `4.4.x`, it seems to work fine. Related: When we move to `clap` version `4.x`, we may need to get rid of the `structopt`. Does that make sense? 

I can take a shot at this.

**Comment by bnjjj** [comment]
Yes it would make sens to get rid of the `structopt` since it's included in clap now
